Fix uuidgen: command not found in v2 this time

FunnyLoaf opened the pull request.

FunnyLoaf committed:
Update codespace-basic-setup.sh

ProgrammerIn-wonderland commented:
I believe auth is deprecated and removed this version, you may want to remove the entire start section

FunnyLoaf commented:
ok got it will do

FunnyLoaf committed:
Update codespace-basic-setup.sh

FunnyLoaf commented:
i have fixed it and removed the start section, but I didn't know how pull requests worked(this is the first one I have ever done besides an accidental commit from my workspace on another project.[which is why i closed and reopened seconds later])

FunnyLoaf committed:
remove start section of codespace-basic-setup.sh
